src/services/generalSearch.service.js

```diff
@@ -0,0 +1,56 @@
+import { authHeader, stringFormat } from '@/helpers';
+import { config } from "@/configurations";
+import axios from "axios";
+
+export const generalSearchService = { searchProfile, searchGroup, searchCase }
+
+function searchProfile(username) {
+    const headers = authHeader();
+    const url = stringFormat(`${config.commonConfig.$apiUrl}/${config.generalSearchConfig.api.searchProfile.endpoint}`, username);
+    return axios
+        .get(
+            url,
+            { headers: headers },
+        )
+        .then((response) => {
+            return response.data;
+        })
+        .catch(() => {
+            throw config.messagingConfig.messages.unknown_error;
```

Comment:
Please add 404 checks in profile and case search services

Reply:
Done Sir